Tests/Recurly/DunningCampaign_Test.php

```diff
@@ -0,0 +1,52 @@
+<?php
+require 'lib/recurly/dunning_campaign.php';
+require 'lib/recurly/dunning_cycle.php';
```

Comment:
I had to include these 2 require statements to get local unit tests running correctly

Reply:
without these 2 require statements i see this error when i run /vendor/bin/phpunit

```
swong-C02YTBPULVCF:recurly-client-php swong$ vendor/bin/phpunit
PHPUnit 7.5.20 by Sebastian Bergmann and contributors.

..R.........R................R....................R.....RRRR...  63 / 196 ( 32%)
...R.....................EEE.............................RR.... 126 / 196 ( 64%)
...........R........................................RRR........ 189 / 196 ( 96%)
R......                                                         196 / 196 (100%)

Time: 157 ms, Memory: 8.00 MB

There were 3 errors:

1) Recurly_DunningCampaignTest::testGetDunningCampaigns
Error: Class 'Recurly_DunningCampaign' not found

/Users/swong/Desktop/recurly_dev_repos/recurly-client-php/Tests/Recurly/DunningCampaign_Test.php:16

2) Recurly_DunningCampaignTest::testBulkUpdateSuccess
Error: Class 'Recurly_DunningCampaign' not found

/Users/swong/Desktop/recurly_dev_repos/recurly-client-php/Tests/Recurly/DunningCampaign_Test.php:24

3) Recurly_DunningCampaignTest::testBulkUpdateFailure
Error: Class 'Recurly_DunningCampaign' not found

/Users/swong/Desktop/recurly_dev_repos/recurly-client-php/Tests/Recurly/DunningCampaign_Test.php:44

--
```

Reply:
runnning `composer dump-autoload` and then re-running the unit tests fixed this issue